apps/glusterfs/cluster_entry_test.go

```diff
@@ -380,3 +380,79 @@ func TestNewClusterEntryNewInfoResponse(t *testing.T) {
 	tests.Assert(t, reflect.DeepEqual(info.Nodes, c.Info.Nodes))
 	tests.Assert(t, reflect.DeepEqual(info.Volumes, c.Info.Volumes))
 }
+
+func TestUpdateClusterEntryForFlags(t *testing.T) {
+	tmpfile := tests.Tempfile()
+	defer os.Remove(tmpfile)
+
+	// Create the app
+	app := NewTestApp(tmpfile)
+	//logger.SetLevel(utils.LEVEL_DEBUG)
+
+	// Create a cluster
+	cluster_req := &api.ClusterCreateRequest{
+		File: true,
```

Comment:
Should we explicitly set `Block: false` here, for clarity?

Reply:
done